Sources/Benchmark/BenchmarkSetting.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public enum BenchmarkSetting {
+    case iterations(Int)
+}
+
+struct BenchmarkSettings {
+    let iterations: Int
+
+    init(_ settings: [[BenchmarkSetting]]) {
+        self.init(Array(settings.joined()))
+    }
+
+    init(_ settings: [BenchmarkSetting]) {
+        var iterations: Int = -1
+
+        for setting in settings {
+            switch setting {
+            case .iterations(let value):
+                iterations = value
+            }
+        }
+
+        self.iterations = iterations
+    }
+}
+
+let defaultSettings: [BenchmarkSetting] = [
```

Comment:
Should this go in constants? Or if not, could you document the policy for what goes where? (FWIW, I think this probably belongs here...)

Reply:
That's a great question. I am dropping Constants file for now, given that it only has a single constant.